pkg/aggregator/mocksender/mocked_methods.go

```diff
@@ -117,6 +117,16 @@ func (m *MockSender) ContainerLifecycleEvent(msgs []serializer.ContainerLifecycl
 	m.Called(msgs)
 }
 
+// ContainerImage submit container image messages
+func (m *MockSender) ContainerImage(msgs []serializer.ContainerImageMessage) {
+	m.Called(msgs)
+}
+
+// SBOM submit container image messages
```

Comment:
Not sure what to put for this one but I believe it's a copy paste ?

```suggestion
// SBOM submit sbom data
```